server/pulp/server/async/scheduler.py

```diff
@@ -192,6 +192,9 @@ def __init__(self, *args, **kwargs):
         self._schedule = None
         self._loaded_from_db_count = 0
 
+        # Setting the celerybeat name
+        self.celerybeat_name = constants.SCHEDULER_WORKER_NAME + "@" + platform.node()
```

Comment:
What about assigning this to a module constant instead of an instance variable? It's never modified so I think a constant would be better.